lib/jsdom/living/form-data.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const formDataSymbols = require("./form-data-symbols");
+
+module.exports = function (core) {
+  const Blob = core.Blob;
+  const File = core.File;
+
+  class FormData {
```

Comment:
Wow, you implemented all of it!!

I think it would be a lot more efficient if you used a Map instead of an array? (I might be missing something.)

You've done enough work already that if you just want to leave a "TODO" comment about Map vs. array that's fine too :)